pkgs/top-level/all-packages.nix

```diff
@@ -10034,6 +10034,8 @@ with pkgs;
 
   rcon = callPackage ../tools/networking/rcon { };
 
+  rcptt = callPackage ../applications/editors/rcptt { };
```

Comment:
If I'm understanding the description of this package correctly, it is not an actual "editor" like, for instance, Vim or Emacs. We probably should move that to some other category then.

Reply:
Okay then, we can leave this as-is.